pr2eus/speak.l

```diff
@@ -24,6 +25,9 @@
       (t
        (send msg :sound sound_play::SoundRequest::*say*)
        (send msg :arg (string sound))))
+    (when (find-method msg :volume)
```

Comment:
@iory can you add comment something like
"since sound_play version x.x.x, the message including volume slot. c.f. github.com/..../pulls/..."

Reply:
OK. I add comment.